pyproject.toml

```
@@ -130,9 +130,9 @@ filterwarnings = [
     # httpx-auth uses datetime.datetime.utcfromtimestamp(), deprecated in Python 3.12
     "ignore:.*Use timezone-aware objects to represent datetimes in UTC.*:DeprecationWarning",
 
-    # 'app' parameter is deprecated in httpx
-    # starlette's TestClient should be updated
-    "ignore:.*'app' shortcut is now deprecated.*:DeprecationWarning",
+    # httpx app is deprecated, use Transport class instead
+    # This should be updated in Starlette's TestClient
+    "ignore:.*shortcut is now deprecated. Use the explicit style.*:DeprecationWarning",
```

Revert unnecessary change
```suggestion
    # 'app' parameter is deprecated in httpx
    # starlette's TestClient should be updated
    "ignore:.*'app' shortcut is now deprecated.*:DeprecationWarning",
```